packages/cxl-ui/scss/cxl-base-card.scss

```diff
@@ -157,8 +157,8 @@ header {
 :host([new]) {
   .badge-new {
     position: absolute;
-    top: calc(-1 * var(--lumo-space-s));
-    right: calc(-1 * var(--lumo-space-s));
+    top: calc(-1 * var(--lumo-space-m));
+    right: 0;
```

Comment:
I see there's difference in Figma design between different cards. I experimented with values and I think original style here makes more sense, so please revert.